src/index.js

```diff
@@ -835,11 +835,15 @@ class ServerlessAppsyncPlugin {
   getGraphQLSchemaResource(config) {
     const logicalIdGraphQLApi = this.getLogicalId(config, RESOURCE_API);
     const logicalIdGraphQLSchema = this.getLogicalId(config, RESOURCE_SCHEMA);
+    const appSyncSafeSchema = config.schema
+      .replace(/"""[^"]*"""\n/g, '') // appsync does not support the new style descriptions
+      .replace(/#.*\n/g, '') // appysnc does not support old-style # comments in enums, so remove them all
+      .replace(/ *& */g, ', '); // appsync does not support the standard '&', but the "unofficial" ',' join for interfaces
```

Comment:
@thomasmichaelwallace what do you think if we throw an error at the plugin level instead of automagically fixing the schema?
Using `.replace()` looks both hacky and dangerous. 

Best practice would probably be to use an appsync-safe schema.

I'm just thinking out loud here. Let me know what you think based on your experience.

Reply:
I guess throwing an error at plugin level if triple quotes, or interface joins with '&'s, etc. addresses the late validation issue (i.e. that the plugin validator lets through app-sync invalid schemas).

I can see why you might not want the replaces in there (I haven't found a library that translates June 2018 spec schemas back down to whatever it is that AWS implemented). That we've been using it for about half a year in production without any issues is probably a bit too much of a "works on my machine ™️" defense.